freud/locality.pyx

```diff
@@ -424,7 +427,8 @@ cdef class NeighborQuery:
                 :meth:`matplotlib.axes.Axes.plot`.
 
         Returns:
-            tuple (:class:`matplotlib.axes.Axes`, :class:`matplotlib.collections.PathCollection`):
+            tuple (:class:`matplotlib.axes.Axes`,
+                   :class:`matplotlib.collections.PathCollection`):
```

Comment:
Sphinx doesn't like this. I'm not sure exactly what it needs. 
![image](https://user-images.githubusercontent.com/3943761/150432364-ac02ad66-400d-4e55-8afa-5b4042813e39.png)